src/detectors/docusaurus.js

```diff
@@ -2,15 +2,25 @@ const { hasRequiredDeps, hasRequiredFiles, getYarnOrNPMCommand, scanScripts } =
 module.exports = function() {
   // REQUIRED FILES
   if (!hasRequiredFiles(['package.json', 'siteConfig.js'])) return false
+
+  const hasV1 = hasRequiredDeps(['docusaurus'])
```

Comment:
Looks like `npx @docusaurus/init@next init my-website bootstrap` doesn't generate a `siteConfig.js` file which means we won't reach this code:
https://v2.docusaurus.io/docs/installation#project-structure
https://docusaurus.io/docs/en/installation#verifying-installation

should we check for `docusaurus.config.js` instead?

Reply:
Nice catch! Updating now!

Reply:
Thanks @monsonjeremy, should we put the config check under the `hasV1` and `hasV2` flags for clarity?

Reply:
Yeah, that's not a bad idea. Let me change that now!